src/Whoops/Handler/XmlResponseHandler.php

```diff
@@ -47,7 +47,7 @@ public function handle()
             ),
         ];
 
-        echo $this->toXml($response);
+        echo static::toXml($response);
```

Comment:
```suggestion
        echo self::toXml($response);
```

?

Reply:
If you use self nobody can extend this function

Reply:
The  function is private

Reply:
I've no preference here, just wanted to close a simple issue that I came across.